   --> ../../third_party/rust/chromium_crates_io/vendor/derivre-v0_3/src/regexbuilder.rs:482:47
    |
482 |             matches!(b, b'\\' | b'"' | 0x7F | 0..0x20)
    |                                               ^^^^^^^
    |
    = note: see issue #37854 <https://github.com/rust-lang/rust/issues/37854> for more information
    = help: add `#![feature(exclusive_range_pattern)]` to the crate attributes to enable
    = help: use an inclusive range pattern, like N..=M

error[E0658]: use of unstable library feature 'char_min'
   --> ../../third_party/rust/chromium_crates_io/vendor/derivre-v0_3/src/syntax.rs:121:37
    |
121 |             && ranges[0].start() == char::MIN
    |                                     ^^^^^^^^^
    |
    = note: see issue #114298 <https://github.com/rust-lang/rust/issues/114298> for more information
    = help: add `#![feature(char_min)]` to the crate attributes to enable


--- a/third_party/rust/chromium_crates_io/vendor/derivre-v0_3/src/lib.rs
+++ b/third_party/rust/chromium_crates_io/vendor/derivre-v0_3/src/lib.rs
@@ -1,3 +1,6 @@
+#![feature(exclusive_range_pattern)]
+#![feature(char_min)]
+
 mod deriv;
 mod hashcons;
 mod nextbyte;
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1109,7 +1109,6 @@ config("compiler") {
 # Don't allow unstable features to be enabled by `#![feature()]` without
 # additional command line flags.
 config("disallow_unstable_features") {
-  rustflags = [ "-Zallow-features=" ]
 }
 
 config("libcxx_hardening") {
